internal/generator/suite.go

```diff
@@ -191,3 +193,68 @@ func (s *Suite) String() string {
 
 	return spaceRegex.ReplaceAllString(strings.TrimSpace(result.String()), "\n")
 }
+
+const bashSuiteTemplate = `
+function setup() {
```

Comment:
Feature request:
I think it would be convenient if bash script printed name of the current suite (ideally full suite path).

Go runner prints a lot of meta info: suite start point, suite name in each line, time in each line.
It would probably be too much work to re-implement all of it in the bash script, but printing the name and start time of each suite setup and cleanup would be convenient but shouldn't be hard to implement.